business_rules/operators.py

```diff
@@ -953,6 +953,7 @@ def target_is_sorted_by(self, other_value: dict) -> pd.Series:
             sort_as: str = col["sort_order"]
             na_pos: str = col["null_position"]
             temp_seseq = self.value.groupby(within).cumcount().apply(lambda x: x + 1)
+            self.value[comparator] = self.value[comparator].apply(lambda x: datetime.fromisoformat(x) if (pd.notnull(x)) else x)
```

Comment:
we cannot assume the comparator is a datetime

Reply:
+ why do we need the datetime at all?

Reply:
Anyway, I think we can process without converting the string into dates because the sorting will work due to the fact that all string dates have the same format with a column.

Reply:
Yes, you are correct, I tried and we can proceed without any conversion into dates . Thanks.